business/products.md

```diff
@@ -0,0 +1,37 @@
+---
+description: Providers of machine translation services
+---
+
+# Providers
+
+Machine translation **providers** offer generic machine translation services with different customization possibilities. Machine translation services have different features and limitations.
```

Comment:
This is kind of hollow.  "different", "various", "several" is often a sign of that.  Anyway, this article isn't really about how the providers offer customisation or differ, so that could go further below, but not in the first sentences.

> Machine translation **providers** offer ready-to-use machine translation systems.  Typically the systems are provided as APIs, and also as applications or integrations.

> They vary by:
> - language support
> - accuracy
> - price
> - customization options
> - data confidentiality
>
> and other features, as well as underlying architecture.